docs/tutorials/rpi.rst

```diff
@@ -0,0 +1,359 @@
+.. _tutorial_rpi:
+
+An offline Raspberry Pi server
+==============================
+
+This guide shows you how to configure a Raspberry Pi as a local WIFI hotspot serving Kolibri.
```

Comment:
Looks like `Wi-Fi`, `Wifi`, `wifi`, or `WiFi` are most common: https://english.stackexchange.com/questions/41180/how-do-you-spell-wifi-wi-fi-wifi

One idea from that thread I like is to call it a "wireless LAN" or WLAN instead.

Reply:
Thanks, fixed :)